recipes/gtk/all/test_package/CMakeLists.txt

```diff
@@ -1,9 +1,7 @@
 cmake_minimum_required(VERSION 3.15)
-project(test_package)
+project(test_package LANGUAGES C)
 
-
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+find_package(gtk REQUIRED CONFIG)
```

Comment:
CMake recommends using pkg_check_modules for GTK:

https://cmake.org/cmake/help/latest/module/FindGTK2.html#examples-finding-gtk-version-3-or-later

The GTK project does not generate .cmake files as well.